openedx/core/djangoapps/theming/management/commands/sites_configuration.py

```diff
@@ -0,0 +1,540 @@
+"""
+Returns the sites' data.
+"""
+
+
+def get_sites_data(dns_name):  # pylint: disable=line-too-long
+    """
+    Return the site configuration of all sites.
+    """
+
+    return {
+        "edx": {
+            "theme_dir_name": "edx.org",
+            "configuration": {
+                "ACTIVATION_EMAIL_FROM_ADDRESS": "edX <activate@edx.org>",
+                "ACTIVATION_EMAIL_SUPPORT_LINK": "https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-",
+                "CREDENTIALS_INTERNAL_SERVICE_URL": "https://credentials-{}.edx.org".format(dns_name),
+                "CREDENTIALS_PUBLIC_SERVICE_URL": "https://credentials-{}.edx.org".format(dns_name),
+                "DISPLAY_ACCOUNT_ACTIVATION_MESSAGE_ON_SIDEBAR": True,
+                "DISPLAY_TOS_IN_MODAL_ON_REGISTRATION_PAGE": True,
+                "ENABLE_COMBINED_LOGIN_REGISTRATION": True,
+                "ENABLE_EXTENDED_COURSE_DETAILS": True,
+                "PASSWORD_RESET_SUPPORT_LINK": "https://support.edx.org/hc/en-us/articles/206212088-What-if-I-did-not-receive-a-password-reset-message-",
+                "COURSE_CATALOG_API_URL": "https://discover-{}.edx.org/api/v1/".format(dns_name),
+                "ENTERPRISE_TAGLINE": " High-quality online learning opportunities from the world's best universities",
+                "REGISTRATION_EXTRA_FIELDS": {
+                    "confirm_email": "hidden",
+                    "country": "required",
+                    "gender": "optional",
+                    "year_of_birth": "optional",
+                    "level_of_education": "optional",
+                    "goals": "optional",
+                    "honor_code": "required",
+                    "terms_of_service": "hidden"
+                },
+                "enable_forum_notifications": True
+            }
+        },
+        "mitxpro": {
```

Comment:
@bill-filler, these configurations are required for the management command to replicate the sites on stage environment, but merging this code into edx-platform does not seem appropriate either.

Reply:
I agree. It might be best to move these configurations into edx-themes in a separate file for each theme.
For example edx-themes/edx-platform/mitxpro.mit.edu/sandbox_config.json.
Then the code in edx-platform should iterate the themes dir and for each load the values in sandbox_config.json if they exist..

Reply:
@afzaledx @hasnain-naveed ^^^